samples/RuleMetadata.sarif

```diff
@@ -0,0 +1,64 @@
+{
+  "$schema": "https://schemastore.azurewebsites.net/schemas/json/sarif-2.1.0-rtm.5.json",
+  "version": "2.1.0",
+  "properties": {
+    "comment": "This sample demonstrates rule metadata and the connection between results and the metadata for the corresponding rules."
+  },
+  "runs": [
+    {
+      "tool": {
+        "driver": {
+          "name": "SarifSamples",
+          "rules": [
+            {
+              "id": "SMP0001",
+              "name": "InvalidUri",
+              "defaultConfiguration": {
+                "level": "error"
+              },
+              "shortDescription": {
+                "text": "URI-valued properties must contain valid URIs."
+               },
+              "fullDescription": {
+                "text": "Every JSON property whose value is defined by the schema to be a URI must contain a valid URI."
+              },
+              "messageStrings": {
+                "default": {
+                  "text": "The URI '{0}' is invalid."
+                }
+              }
+            }
+          ]
+        }
+      },
+      "results": [
+        {
+          "properties": {
+            "comment": "The ruleIndex property points into the array tool.driver.rules."
+          },
+          "ruleId": "SMP0001",
+          "level": "error",
+          "ruleIndex": 0,
+          "message": {
+            "text": "The URI '://C:/code/dev' is invalid."
```

Comment:
>text [](start = 13, length = 4)

Why aren't you using message arguments here? Isn't the point of this sample to help verify rule metadata is properly retrieved? Omitting the full plaintext message will help, because then it becomes very clear the viewer hasn't rendered the message properly.